packages/cli/openapi-ir-to-fern/src/buildFernDefinition.ts

```diff
@@ -46,26 +46,31 @@ export function buildFernDefinition(context: OpenApiIrConverterContext): FernDef
     }
 
     // Add Schemas
-    for (const [id, schema] of Object.entries(context.ir.schemas)) {
-        if (schemaIdsToExclude.includes(id)) {
-            continue;
-        }
+    for (const [namespace, schemas] of Object.entries(context.ir.schemas)) {
+        for (const [id, schema] of Object.entries(schemas)) {
+            if (schemaIdsToExclude.includes(id)) {
+                continue;
+            }
 
-        const declarationFile = getDeclarationFileForSchema(schema);
-        const typeDeclaration = buildTypeDeclaration({ schema, context, declarationFile });
+            const declarationFile = getDeclarationFileForSchema(schema);
+            const typeDeclaration = buildTypeDeclaration({ schema, context, declarationFile, namespace });
```

Comment:
This is really only nesting the for loop into another for loop, the only actual change is passing in namespace here